Solutions/CiscoDuoSecurity/Hunting Queries/CiscoDuoUnpachedAccessDevices.yaml

```diff
@@ -0,0 +1,30 @@
+id: 9de62fee-f601-43c9-8757-2098e59fedeb
+name: Cisco Duo - Devices with vulnerable OS
+description: |
+  'Query searches for devices with vulnerable OS.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: CiscoDuoSecurity
+    dataTypes:
+      - CiscoDuo
+tactics:
+  - InitialAccess
+relevantTechniques:
+  - T1078
+query: |
+  let os_latest = 'x.x.xxx'

```

Comment:
Might be a good idea to put a comment that we need to put the latest Os version before running the query.

Reply:
Done by the author.